### PR TITLE
Chris seo enhancement

### PR DESCRIPTION
**Pull request by eahk**

### CODE REVIEW

**Comment by eahk** [comment]
Screenshot 2021-07-19 at 15 48 38
Unexpected facebook iframe space

Screenshot 2021-07-07 at 14 27 55
Unexpected header tracking tag space

File: frontend/js/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ import backLinks from "./components/back-links";
 import parallax from "./components/parallax";
 import scrollWay from "./components/scroll-way";
 import articleList from "./components/article-list";
-import facebookVideo from "./components/facebook-video";
+// import facebookVideo from "./components/facebook-video";
```
Comment (hi-upchen) [inline frontend/js/index.js:32]: If a code is useless, then remove it. 
Or add a comment about when to un-comment this code.

File: frontend/scss/_utilities.scss [hunk 2]
```diff
@@ -17,3 +17,19 @@
   overflow: hidden;
   clip: rect(1px, 1px, 1px, 1px);
 }
+
+html > iframe {
```
Comment (hi-upchen) [inline frontend/scss/_utilities.scss:21]: `html, body > iframe`

Comment (hi-upchen) [reply frontend/scss/_utilities.scss:21]: Add a comment about why there's such css rule here.

**Review by hi-upchen — APPROVED** [review]

**Pull request merged**